src/osdc/Objecter.cc

```diff
@@ -2460,6 +2460,10 @@ void Objecter::_op_submit(Op *op, shunique_lock& sul, ceph_tid_t *ptid)
   sl.unlock();
   put_session(s);
 
+ if ((cct->_conf->rgw_fastfail_homeless_timeout > 0) && s->is_homeless()) { 
```

Comment:
Objecter is used by all librados clients, so shouldn't refer to any rgw-specific config variables. if anything, this should be named objecter_homeless_timeout